packages/upload-client/test/store.test.js

```diff
@@ -348,21 +347,111 @@ describe('Store.list', () => {
     assert(service.store.list.called)
     assert.equal(service.store.list.callCount, 1)
 
-    assert.equal(list.count, res.count)
-    assert.equal(list.page, res.page)
-    assert.equal(list.pageSize, res.pageSize)
+    assert.equal(list.cursor, res.cursor)
+    assert.equal(list.size, res.size)
     assert(list.results)
     assert.equal(list.results.length, res.results.length)
     list.results.forEach((r, i) => {
-      assert.equal(
-        r.payloadCID.toString(),
-        res.results[i].payloadCID.toString()
-      )
+      assert.equal(r.payloadCID, res.results[i].payloadCID)
       assert.equal(r.size, res.results[i].size)
       assert.equal(r.uploadedAt, res.results[i].uploadedAt)
     })
   })
 
+  it('paginates', async () => {
+    const cursor = 'test'
```

Comment:
nit: `cursor` is `payloadCID` of latest entry with Dynamo. Maybe better to do that in the test?